app/models/course/announcement.rb

```diff
@@ -18,4 +19,19 @@ def set_defaults
   def valid_in_future?
     valid_from > DateTime.now
   end
+
+  # Get the number of unread announcements
```

Comment:
You should document this function properly, see the documentation of other existing code